src/clr-core/styles/typography/typography.stories.ts

```diff
@@ -73,6 +73,21 @@ export const weights = () => {
   `;
 };
 
+export const sizes = () => {
+  return html`
+    <cds-demo cds-layout="vertical gap:md">
+      <p cds-text="body" style="font-size: var(--cds-token-typography-font-size-0)">The quick brown foxes jumps over the lazy dog.</p>  
+      <p cds-text="body" style="font-size: var(--cds-token-typography-font-size-1)">The quick brown foxes jumps over the lazy dog.</p>
+      <p cds-text="body" style="font-size: var(--cds-token-typography-font-size-2)">The quick brown foxes jumps over the lazy dog.</p>
+      <p cds-text="body" style="font-size: var(--cds-token-typography-font-size-3)">The quick brown foxes jumps over the lazy dog.</p>
+      <p cds-text="body" style="font-size: var(--cds-token-typography-font-size-4)">The quick brown foxes jumps over the lazy dog.</p>
```

Comment:
Are we intending for end users to use these? It seems like a step back to me.

Reply:
Removing not meant to be public